.gitignore

```diff
@@ -16,6 +16,9 @@
 !/pootle/translations/tutorial/
 !/pootle/translations/templates/
 
+# /pootle/pootle_fs/
+/pootle/pootle_fs/*/
```

Comment:
i think this needs to be called pootle_fs_working or similar and be more hidden - fail here is users a) touching the files b) adding stuff here